sky/packages/sky/lib/sky_tool

```diff
@@ -29,6 +30,9 @@ APK_NAME = 'SkyShell.apk'
 ANDROID_PACKAGE = "org.domokit.sky.shell"
 ANDROID_COMPONENT = '%s/%s.SkyActivity' % (ANDROID_PACKAGE, ANDROID_PACKAGE)
 
+SKY_SHELL_APP_ID = 'com.google.SkyShell'
```

Comment:
Does that mean people who don't work for Google can't produce a build of SkyShell for iOS using open source code?